string NAME=land

ObjectDrone d
